migrations/versions/405672cf4046_reports.py

```diff
@@ -24,7 +24,8 @@ def upgrade() -> None:
         CREATE TABLE reports (
             session_id varchar REFERENCES demo_sessions,
             target_steam_id varchar,
-            created_at timestamptz
+            created_at timestamptz,
+            PRIMARY KEY (session_id, target_steam_id)
```

Comment:
Could/should this be versioned separately under `alembic` or what would be the ops overhead of that

Reply:
It could be for sure, but nobody is using this table yet so I will just manually downgrade and upgrade with the new primary key